autoquran/src/main/res/values/sura_names.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
can we not share the existing one? I understand because it's in `app` now, but we can move it (and the icons) to a `common` module and share it?

Reply:
I already did this in my branch.